deployment/kustomizations/examples/secret-wren_example.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: wrenai-secrets
+type: Opaque
+data:
+  # OPENAI_API_KEY is REQUIRED: without a valid key the wren-ai-service-deployment pod will not start
+  OPENAI_API_KEY: UkVRVUlSRUQ6IHNrLXByb2otYWxsLWFjY2Vzcy1wbGFjZWhvbGRlci00LXdyZW4tYWktc2VydmljZS1kZXBsb3ltZW50
+  # OPENAI_BASE_URL is not implemented yet to use other OpenAI API-compatible endpoints
+  OPENAI_BASE_URL: aHR0cHM6Ly9hcGkub3BlbmFpLmNvbS92MQ==
+  # PG_HOST is not implemented yet
+  PG_HOST: bG9jYWxob3N0
+  # PG_PORT is not implemented yet
+  PG_PORT: NTQzMg==
+  PG_USERNAME: cG9zdGdyZXM=
+  PG_URL: cG9zdGdyZXM6Ly9wb3N0Z3Jlczpwb3N0Z3Jlc0Bsb2NhbGhvc3Q6NTQzMi9hZG1pbl91aQ==
+  # PG_DATABASENAME is not implemented yet
+  PG_DATABASENAME: cG9zdGdyZXM=
+  POSTHOG_API_KEY: cGhjX2tleS1wbGFjZWhvbGRlcg==
+  POSTHOG_HOST: aHR0cHM6Ly9hcHAucG9zdGhvZy5jb20=
+  QDRANT_HOST: d3Jlbi1xZHJhbnQ=
+  # QDRANT_PORT is not implemented yet
+  QDRANT_PORT: NjMzMw==
+  # QDRANT_COLLECTION_NAME is not implemented yet
+  QDRANT_COLLECTION_NAME: d3Jlbi1jb2xsZWN0aW9u
+  # QDRANT_DIMENTIONS is not implemented yet. 
+  # Depends on your embeddings model: embedding vector will be 1536 for text-embedding-3-small or 3072 for text-embedding-3-large
+  QDRANT_DIMENTIONS: MzA3Mg==
+  # QDRANT_HTTP_MODE is not implemented yet: http vs. https
+  QDRANT_HTTP_MODE: http
+  # QDRANT_API_KEY is not implemented yet
+  QDRANT_API_KEY: cWRyYW50LWFwaS1rZXktcGxhY2Vob2xkZXI=
+  USER_UUID: dXNlci1wbGFjZWhvbGRlcg==
```

Comment:
We could remove following properties because they're not implemented.
* OPENAI_BASE_URL
* PG_HOST
* PG_PORT
* PG_DATABASENAME
* QDRANT_COLLECTION_NAME
* QDRANT_DIMENTIONS
* QDRANT_HTTP_MODE
* QDRANT_API_KEY

We could move following into configmap, IMO
* PG_USERNAME
* POSTHOG_HOST
* QDRANT_HOST
* QDRANT_PORT
* USER_UUID, this value is a UUID format, could set to `00000000-0000-0000-0000-000000000000` as default value in configmap.

Reply:
I would've keep it so you would not forget to implement these at some point. Its important in k8s environment @wwwy3y3

Reply:
* We use `PG_URL` instead of `PG_HOST`, `PG_PORT`, `PG_USERNAME`, & `PG_DATABASENAME`, so you could remove these variables.
* `QDRANT_*` variables: check the comment here -> https://github.com/Canner/WrenAI/pull/318/files/9c238f062a4a0a5427d5692328b873ae71db326f#r1620870185. I'm not really sure if they all make sense.
* Still think `POSTHOG_HOST`, `USER_UUID` do not count as credentials, should be in configmap instead of secrets.

Reply:
The reason why I'm keeping these in the Secrets because i do not want to split a config setting to the same thing into two separate manifests, so if we have an API key for POSTHOG_API_KEY then we want to store it in Secrets, then POSTHOG_HOST makes sense to also keep it in Secrets next to POSTHOG_API_KEY so users would not be jumping between ConfigMap and Secrets manifest to configure one thing, which is in this example POSTHOG. Hoe it makes sense. @wwwy3y3

Reply:
Got it. Thanks for the clarification.